styles/website.css

```diff
@@ -112,3 +112,13 @@ section.markdown-section table {
   display: block;
   overflow-x: auto;
 }
+
+.book .book-summary ul.summary > li[data-level="1.2"] > ul > li > a {
```

Comment:
I'll have to check these out. I had added some styles already so these might be duplicate.